
# HG changeset patch
# User Sam Lantinga <slouken@libsdl.org>
# Date 1255169641 0
# Node ID 8ec3036098dfd648a492a4a9e2c692c309188239
# Parent  f2f8525e60c21beefa8956e43a07ea2f5c12bc33
Adapted from Debian patch: 320_activate_xrandr_on_default.diff

diff -r f2f8525e60c2 -r 8ec3036098df src/video/x11/SDL_x11modes.c
--- a/src/video/x11/SDL_x11modes.c	Sat Oct 10 10:10:36 2009 +0000
+++ b/src/video/x11/SDL_x11modes.c	Sat Oct 10 10:14:01 2009 +0000
@@ -421,11 +421,16 @@
         return 0;
     }
 
-    /* This defaults off now, due to KDE window maximize problems */
-    if ( !env ) {
+    /* This used to default off, due to KDE window maximize problems */
+    /* Reactivated since I haven't encountered such problems with KDE, but if
+       one does encounter such problems he/she can just set
+       SDL_VIDEO_X11_XRANDR to 0
+       Closes Debian bug: #450689
+     */
+/*  if ( !env ) {
         return 0;
     }
-
+*/
     if ( !SDL_X11_HAVE_XRANDR ) {
         return 0;
     }

